charts/ztka/README.md

```diff
@@ -1,14 +1,69 @@
 # ztka
 
+A Helm chart for Paralus ZTKA.
+
 ![Version: 0.1.8](https://img.shields.io/badge/Version-0.1.8-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.1.7](https://img.shields.io/badge/AppVersion-v0.1.7-informational?style=flat-square)
 
-A Helm chart for Paralus ZTKA.
+This chart bootstraps the Paralus deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes v1.18+
+
+## Get Repo Info
+
+```console
+helm repo add paralus https://paralus.github.io/helm-charts
+helm repo update
+```
+
+## Install Chart
+
+**Important:** only helm3 is supported
+
+```console
+helm install [RELEASE_NAME] paralus/ztka -n paralus  \
+--create-namespace \
+--set deploy.postgresql.enable=true
+```
+
+The command deploys Paralus on the Kubernetes cluster in the default configuration.
+
+_See [configuration](#configuration) below._
+
+_See [helm install](https://helm.sh/docs/helm/helm_install/) for command documentation._
+
+## Uninstall Chart
+
+```console
+helm uninstall [RELEASE_NAME]
+```
+
+This removes all the Kubernetes components associated with the chart and deletes the release.
+
+_See [helm uninstall](https://helm.sh/docs/helm/helm_uninstall/) for command documentation._
+
+## Upgrading Chart
+
+```console
+helm upgrade [RELEASE_NAME] paralus/ztka --install
+```
+
+_See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
+
+### Upgrading to chart v0.1.9
+
+The chart `v0.1.9` introduced support for both database and elasticsearch as a back-end for auditlog storage. So upgrading from older version to v0.1.9 may affect the auditing functionality.
+
+_It is recommended to keep using either database or elasticsearch for auditing storage to avoid auditlog data loss throughout upgrades._
```

Comment:
let's say user installed paralus with es before v0.1.9 and wants to upgrade, let's add the upgrade command here as well

helm upgrade [RELEASE_NAME] paralus/ztka --install -n paralus --set auditLog.storage="elasticsearch"

deploy elasticsearch, filebeat values would already be set I suppose

Reply:
> deploy elasticsearch, filebeat values would already be set I suppose

It would reset to values to the one built into the chart. So advised to use `values.elasticsearch.yaml` that has all elasticsearch configuration.